File: KotlinPractice/app/src/main/java/com/ryadar/kotlinpractice/MyHolder.java
package com.ryadar.kotlinpractice;

import android.support.v7.widget.AppCompatImageView;
import android.support.v7.widget.RecyclerView;
import android.view.View;

class MyHolder extends RecyclerView.ViewHolder {
     MyView imageView;
    public MyHolder(View itemView) {
        super(itemView);
        imageView=itemView.findViewById(R.id.imageView);

    }
}

File: KotlinPractice/app/src/main/java/com/raja/kotlin/raja/kotlin/android/MyHolder.kt
package com.raja.kotlin.raja.kotlin.android

import android.support.v7.widget.RecyclerView
import android.view.View
import android.widget.TextView
import com.ryadar.kotlinpractice.R

class MyHolder(itemView: View) : RecyclerView.ViewHolder(itemView) {
    var myItem:View
    init {
        myItem=itemView
    }
    var name:TextView =myItem.findViewById(R.id.name);



}